deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxAttributesFinderFeature.java

```diff
@@ -108,15 +133,31 @@ public PathAttributes toAttributes(final DeepBox deepBox) {
         return attrs;
     }
 
-    @Override
-    public PathAttributes toAttributes(final Node node) {
+
+    public PathAttributes toAttributes(final Node node) throws ApiException {
         final PathAttributes attrs = new PathAttributes();
         attrs.setFileId(node.getNodeId().toString());
         attrs.setCreationDate(node.getCreated().getTime().getMillis());
         attrs.setModificationDate(node.getModified().getTime().getMillis());
+        attrs.setSize(node.getSize());
+        // TODO check with DeepBox/yla: which URL to provide? only share link / resource url?
+        attrs.setLink(new DescriptiveUrl(URI.create(new HostUrlProvider()
```

Comment:
No link provided by API therefore move this to `DeppboxUrlProvider`.  Return of type `DescriptiveUrl.Type.http` to allow opening in web browser.